src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -155,6 +155,23 @@ export class DictionaryConcepts extends Component {
     return matchSorter(rows, filter.value, { keys: [id] });
   };
 
+  handleConcepts = (concepts) => {
```

Comment:
This code snippet assumes there is only going to be two concept sources ('CIEL' and 'custom'). Can this be modified to generically be concept.source, and only modify known concepts without friendly source name. 
```
concepts.forEach((concept) => {

      if (concept.source === 'something that identifies custom source') {
        const newConcept = {
          ...concept,
          source: 'Custom',
        };
            newConcepts.push(newConcept);
      }else{
           newConcepts.push(concept);
      }
```

Reply:
@ozezzy , currently in the whole application, a concept is either `CIEL` or `custom`. We have no other sources.

Reply:
@Duncanian that means this will need to be modified when any other other source is added ?

Reply:
Yeah, and that will have to be in the post MVP when we will have `PIH concepts`

Reply:
ok

Reply:
ok